test/extensions/filters/http/compressor/compressor_filter_integration_test.cc

```diff
@@ -70,11 +70,26 @@ class CompressorIntegrationTest : public testing::TestWithParam<Network::Address
       name: compressor
       typed_config:
         "@type": type.googleapis.com/envoy.extensions.filters.http.compressor.v3.Compressor
-        disable_on_etag_header: true
-        content_length: 100
-        content_type:
-          - text/html
-          - application/json
+        response_direction_config:
+          disable_on_etag_header: true
+          remove_accept_encoding_header: false
+          common_config:
+            enabled:
+              default_value: true
+              runtime_key: foo_key
+            min_content_length: 100
+            content_type:
+              - text/html
+              - application/json
+        request_direction_config:
+          common_config:
+            enabled:
+              default_value: false
+              runtime_key: enable_requests
```

Comment:
This means that the request path isn't tested in this integration test, should it be tested as well?

Reply:
Right, I missed that. Added a case for compressed requests.

Currently we test 1) compressed request only, 2) compressed response only, 3) neither request nor response. Here we are missing a case for compressed request and response. But for this the test code should be better refactored into table-based tests IMHO. Can we do that in a follow-up PR or in this PR?

Reply:
SGTM as a follow-up PR.